src/app/onprem/license-nag-message-management/nag-message.html

```diff
@@ -0,0 +1,20 @@
+<!--
+
+    Copyright (c) 2015-2016 Codenvy, S.A.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Codenvy, S.A. - initial API and implementation
+
+-->
+<div class="nag-message">
```

Comment:
maybe classname should include license to avoid collision with another "nag-message"